tools/README.md

```diff
@@ -0,0 +1,17 @@
+# A collection of Sync Tools and utilities
+
+See each directory for details:
+
+* [hawk](hawk) - a tool for generating test HAWK authorization headers
+* [spanner](spanner) - Google Cloud Platform Spanner tools for maintenance and testing
+* [user_migration](user_migration) - scripts for dumping and moving user data from SQL to Spanner
+
+## Installation
+
+These tools are mostly written in python. It is recommended that you create a commonly shared virtual environment using something like:
+
+`python3 -m venv venv`
```

Comment:
should make it clear here and probably in the other README that user_migration is python2

Reply:
`migrate_user.py` can run under Python3. the `dump_*.py` however can't because of a requirement in arvo.